gnmpagerduty/__init__.py

```diff
@@ -0,0 +1,8 @@
+from plugin import *
+from tasks import check_storage
+
+__version__ = 'DISPLAY_VERSION'
+__production__ = False
+
+check_storage.delay()
```

Comment:
I have rewritten the code for setting up the task as suggested and it is now working on the dev. system.